src/index.html

```diff
@@ -4,9 +4,277 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <div class="wrapper">
+      <header class="header">
+        <div class="header__container">
+          <a href="#" class="header__logo">
+            <img
+              src="./images/logo.png"
+              alt="logo"
+              class="header__img"
+            >
+          </a>
+
+          <nav class="header__menu menu">
+            <ul class="menu__list">
+              <li class="menu__item">
+                <a href="#" class="menu__link is-active"> Apple</a>
+              </li>
+              <li class="menu__item">
```

Comment:
```suggestion
              </li>
              
              <li class="menu__item">
```

Add one empty line between sibling blocks inside the same parent block for both of them.